# awesome-expression-browser

[![Build Status](https://travis-ci.org/federicomarini/awesome-expression-browser.svg?branch=master)](https://travis-ci.org/federicomarini/awesome-expression-browser)

A curated list of software and resources for exploring and visualizing (browsing) expression data, but not only limited to that. Credits for the backbone of the structure go to Sean Davis and his [awesome-single-cell](https://github.com/seandavi/awesome-single-cell) repository.

[Contributions welcome](https://github.com/federicomarini/awesome-expression-browser/blob/master/contributing.md)!

## Software list

- [adultHSPC10X](https://gottgens-lab.stemcells.cam.ac.uk/adultHSPC10X/) as a companion to http://www.bloodjournal.org/content/early/2018/03/23/blood-2017-12-821413 
- [Allen Brain Atlases](http://portal.brain-map.org)
- [ASAP](https://asap.epfl.ch)
- [BioTuring Browser](http://bioturing.com/product/bbrowser) 
- [cBioPortal](https://www.cbioportal.org)
- [cellxgene](https://github.com/chanzuckerberg/cellxgene), deployed e.g. for the Tabula Muris (http://tabulamuris.cxg.czi.technology) - a simpler browser is available at https://tabula-muris.ds.czbiohub.org 
- [DRSC RNA-seq Explorer](https://www.flyrnai.org/tools/rna_seq/web/showProject/1) Lab website: https://www.flyrnai.org/scRNA/ https://www.biorxiv.org/content/10.1101/410423v1 + 
- [DVEX](https://shiny.mdc-berlin.de/DVEX/) accompanying http://science.sciencemag.org/content/358/6360/194 (www.dvex.org)
- [epiviz](http://epiviz.cbcb.umd.edu), also accessible via epivizr (http://bioconductor.org/packages/release/bioc/html/epivizr.html)
- [Expression Atlas](https://www.ebi.ac.uk/gxa/home) + https://www.ebi.ac.uk/gxa/sc/home
- [FireBrowse](http://firebrowse.org)
- [GenePattern](http://genepattern-notebook.org)
- [GTEx Portal](https://gtexportal.org/home/) Good olâ€™ GTEx portal: 
- [Haemosphere](https://www.haemosphere.org/) - I reviewed that :)
- [HCA Bone Marrow Viewer](http://www.altanalyze.org/ICGS/HCA/Viewer.php), related to this publication https://doi.org/10.1016/j.exphem.2018.09.004
- [HTSVis](http://htsvis.dkfz.de/HTSvis/), for image screening. Code available at https://github.com/boutroslab/HTSvis, described in https://academic.oup.com/bioinformatics/article/33/18/2960/3827333
- [ideogram.js](https://github.com/NCBI-Hackathons/rnaseqview) described in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5605993/
- [iS-CellR](https://github.com/immcore/iS-CellR), for analysing and visualising scRNA-seq data. R Shiny app. Manuscript available at https://academic.oup.com/bioinformatics/article/34/24/4305/5048937
- [iSEE](https://bioconductor.org/packages/release/bioc/html/iSEE.html) - Interactive SummarizedExperiment Explorer (dev version at https://github.com/csoneson/iSEE), with custom panels (https://github.com/kevinrue/iSEE_custom). Publication available at https://f1000research.com/articles/7-741/v1
- [Loupe Cell Browser](https://support.10xgenomics.com/single-cell-gene-expression/software/visualization/latest/what-is-loupe-cell-browser), from 10x Genomics
- [Lung Aging Atlas](http://146.107.176.18:3838/MLAA_backup/), reachable from https://theislab.github.io/LungAgingAtlas/ and linked to this publication: https://www.nature.com/articles/s41467-019-08831-9
- [Mammary Gland Development](https://marionilab.cruk.cam.ac.uk/mammaryGland/), companion to https://www.nature.com/articles/s41467-017-02001-5
- [Morpheus](https://software.broadinstitute.org/morpheus/)
- [Mouse Cell Atlas](http://bis.zju.edu.cn/MCA/) Microwell-seq, companion to https://doi.org/10.1016/j.cell.2018.02.001
- [Mouse Gastrulation and Early Organogenesis](https://marionilab.cruk.cam.ac.uk/MouseGastrulation2018/), accompanying https://www.nature.com/articles/s41586-019-0933-9
- [Mouse Organogenesis](https://marionilab.cruk.cam.ac.uk/organogenesis/), companion to https://www.nature.com/articles/s41556-017-0013-z
- [Mouse Organogenesis Cell Atlas (MOCA)](https://oncoscape.v3.sttrcancer.org/atlas.gs.washington.edu.mouse.rna/landing), companion to the manuscript https://www.nature.com/articles/s41586-019-0969-x
- [Mouse sci-ATAC-seq Atlas](http://atlas.gs.washington.edu/mouse-atac/), for example http://atlas.gs.washington.edu/mouse-atac/ukbb_visualization/, among others. Manuscript: https://doi.org/10.1016/j.cell.2018.06.052
- [Multiple sclerosis gene atlas](http://msatlas.compbio.sdu.dk)
- [Planaria Single Cell Atlas](https://shiny.mdc-berlin.de/psca/) 
- [Planarian Digiworm](https://digiworm.wi.mit.edu), companion to http://science.sciencemag.org/content/360/6391/eaaq1736.full
- [pRolocGUI](http://bioconductor.org/packages/release/bioc/html/pRolocGUI.html) (for proteomics)
- [Reptilian pallium](https://public.brain.mpg.de/shiny/apps/SingleCellTranscriptomics/) More exotic species: https://brain.mpg.de/research/laurent-department/software-techniques.html
- [scClustViz](https://baderlab.github.io/scClustViz/) - focused on exploring clustering results, paper here: https://f1000research.com/articles/7-1522/v2
- [scSVA](https://github.com/klarman-cell-observatory/scSVA), single-cell Scalable Visualization and Analytics
- [Seattle Organismal Molecular Atlases](http://atlas.gs.washington.edu/hub/)
- [shinyMethyl](https://www.bioconductor.org/packages/release/bioc/html/shinyMethyl.html) (for methylation)
- [Single Cell Portal](https://portals.broadinstitute.org/single_cell) at the Broad Institute; example here: https://portals.broadinstitute.org/single_cell/study/SCP43/atlas-of-human-blood-dendritic-cells-and-monocytes#study-visualize - somehow takes very long to load?
- [TCGABrowser](http://tcgabrowser.ethz.ch:3838/PROD/)
- [WIlsON](https://cran.r-project.org/web/packages/wilson/), described in https://academic.oup.com/bioinformatics/article/35/6/1055/5078467; demo here http://loosolab.mpi-bn.mpg.de
- [Xena Browser](https://xenabrowser.net) ucsc/xena
- [Zebrafish single cell development atlas](https://kleintools.hms.harvard.edu/paper_websites/wagner_zebrafish_timecourse2018/mainpage.html)


## Other resources:

- A collection of tools related to the Human Cell Atlas project: https://prod.data.humancellatlas.org/analyze
- An overview/comment on some of the above: http://science.sciencemag.org/content/358/6360/172
- A number of resources can be found at http://www.teichlab.org/data, although the newer sets adopted the use of iSEE
- Something merfish-specific?

